app/views/groups/members/index.html.erb

```diff
@@ -23,10 +26,29 @@
   <% end %>
 <% end %>
 
-<div class="bg-white dark:bg-slate-800">
-  <%= turbo_frame_tag "new_member_modal" %>
-  <%= turbo_frame_tag "member-update-alert" %>
+<div class="flex flex-row-reverse">
+  <%= search_form_for @q, url: group_members_url(**request.query_parameters), html: {"data-controller": "filters"} do |f| %>
```

Comment:
The search form breaks if you are on the `Groups` tab and try to do a search. If you clear the text box the table for members is displayed but the `Groups` tab is selected.

I think it may be good to create a members search component which will be used both by the members and groups tabs for both groups and projects